packages/components/src/rating/index.js

```diff
@@ -72,6 +73,14 @@ Rating.propTypes = {
 	 * Additional CSS classes.
 	 */
 	className: PropTypes.string,
+	/**
+	 * Icon used, defaults to StarIcon
+	 */
+	icon: PropTypes.elementType,
+	/**
+	 * Outline icon used, the not selected rating. Defaults to props.icon or StarIcon
+	 */
+	outlineIcon: PropTypes.elementType,
 };
```

Comment:
This allows me to use the filled star and the outline star:
<img width="116" alt="Screen Shot 2020-11-23 at 4 24 18 PM" src="https://user-images.githubusercontent.com/2240960/100011619-5c3a1c80-2da8-11eb-8c5d-98ba20d8bd9d.png">